src/index.ts

```diff
@@ -105,6 +107,7 @@ export {
   ExitError,
   IPromptOptions,
   Table,
+  SuperTable,
```

Comment:
you're exporting implementation here which is going to cause `./styled/supertable` to always be loaded. I would either take out the lazy loading or make it so you only export types